scripts/install-helm.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash -e
+
+#Based on ideas from https://github.com/rook/rook/blob/master/tests/scripts/helm.sh
+
+temp="/tmp/ceph-csi-tests-scripts-helm"
+
+HELM="helm"
+HELM_VERSION=${HELM_VERSION:-"v2.16.5"}
+arch="${ARCH:-}"
+
+CEPHFS_CHART_NAME="ceph-csi-cephfs"
+RBD_CHART_NAME="ceph-csi-rbd"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+detectArch() {
+    case "$(uname -m)" in
+    "x86_64" | "amd64")
+        arch="amd64"
+        ;;
+    "aarch64")
+        arch="arm64"
+        ;;
+    "i386")
+        arch="i386"
+        ;;
+    *)
+        echo "Couldn't translate 'uname -m' output to an available arch."
+        echo "Try setting ARCH environment variable to your system arch:"
+        echo "amd64, x86_64. aarch64, i386"
+        exit 1
+        ;;
+    esac
+}
+
+install() {
+    if ! helm_loc="$(type -p "helm")" || [[ -z ${helm_loc} ]]; then
+        # Download and unpack helm
+        local dist
+        dist="$(uname -s)"
+        # shellcheck disable=SC2021
+        dist=$(echo "${dist}" | tr "[A-Z]" "[a-z]")
+        mkdir -p "${temp}"
+        wget "https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-${dist}-${arch}.tar.gz" -O "${temp}/helm.tar.gz"
+        tar -C "${temp}" -zxvf "${temp}/helm.tar.gz"
+        HELM="${temp}/${dist}-${arch}/helm"
+    fi
+
+    # set up RBAC for helm
+    kubectl --namespace kube-system create sa tiller
```

Comment:
Just wondering, why `tiller`?

Reply:
we need a tiller to install charts. https://v2.helm.sh/docs/install/#installing-tiller